### ISSUE TITLE
[local build] Incomplete executable path upon build completion

### ISSUE THREAD

**Issue by sandordargo** [tipi-build/cli#26]
When I build something local, I get a build summary.
For example:

```
  executables
  ===========

    * 2021/OOP-Paths-Solution-W-Doxygen/OOP-gyak-path-solution
```

The mentioned path is incomplete, in should be preprended with `./build/<env>/bin/`, in this case with `./build/linux-cxx17/bin/`

**Comment by daminetreg** [comment]
Yes indeed it's the path of the source file without the extension, would be better to show the destination.

**Comment by pysco68** [comment]
One more issue we think we've solved nicely in v0.0.32:

![grafik](https://user-images.githubusercontent.com/90577/175088616-dfe234eb-ec72-415d-87d1-5fe07f86a78f.png)

Looking forward to your feedback

**Comment by sandordargo** [comment]
I just tried it and this is absolutely awesome! Thanks a lot!
